source/test/default_mode_test/SymWorld.test.cc

```diff
@@ -960,63 +837,64 @@ TEST_CASE( "Host Phylogeny", "[default]" ){
       size_t taxon_info = host_sys->GetTaxonAt(pos)->GetInfo();
       size_t expected_taxon_info = 10;
 
-      THEN("the occupying organism is removed from the systematic"){
+      THEN("the organism is tracked by the systematic"){
         REQUIRE(w.GetNumOrgs() == 1);
         REQUIRE(host_sys->GetNumActive() == 1);
         REQUIRE(expected_taxon_info == taxon_info);
       }
     }
 
+
     //ORGANISMS ADDED TO TAXA
```

Comment:
This set of code could probably be in a for loop to reduce the amount of repeated lines that are really similar but checking different taxa?